public/redux/reducers/mapper.ts

```diff
@@ -54,6 +54,18 @@ export function getTypeFromMappings(
   } else {
     currentDataTypes[type] = [path];
   }
+
+  if (mappings.fields) {
+    Object.entries(mappings.fields).forEach(([field, value]) => {
+      currentDataTypes = getTypeFromMappings(
+        value as Mappings,
+        currentDataTypes,
+        resolvePath(path, field)
+      );
+    });
+    return currentDataTypes;
```

Comment:
minor: this `return` might not be necessary as next line is the same.

Reply:
Good catch, thanks.